docs/Cli.md

```diff
@@ -133,6 +133,7 @@ Re-apply any new defaults as desired.
 | `auto_disarm_delay`             |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | 0      | 60     | 5             | Master       | UINT8    |
 | `small_angle`                   | If the copter tilt angle exceed this value the copter will refuse to arm. default is 25°.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | 0      | 180    | 25            | Master       | UINT8    |
 | `pid_at_min_throttle`           | If enabled, the copter will process the pid algorithm at minimum throttle.  Cannot be used when `retarded_arm` is enabled.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | OFF    | ON     | ON            | Master       | UINT8    |
+| `airmode_saturation_limit`      | When air mode is enabled this will act as a "crash detection". It prevents too much compensation for very high errors what can cause spazzing out. 0 = no limit 100 is always limited during saturation.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | 0      | 100    | 50            | Master       | UINT8    |
```

Comment:
Was thinking about something like that, but decided to fit it in a 8bit integer. It is really only meant to guard against crashes. I don't think people will use this parameter that much at all. Folks do not really need to know detaills behind it.

Anyway something else I found out.Has nothing to do with this PR or part of code,but is something I found out during extreme testing.
I discovered that gyroADC has no limit. That means that when quadcopter configured in certain way with very high rates can cause inverted gyro data. I managed to get that in the log as well. So above 2000deg/sec gyro goes completaly opposite what can result in infinite rolls. Where do you think it would be best way to protect that.I would say each PID controller should do some kind of limiting?